File: Java/src/main/java/at/nagarro/exercises/bank/AccountReport.java
package at.nagarro.exercises.bank;

import java.math.BigDecimal;
import java.time.LocalDate;

public class AccountReport {
    public Money balance;
    public String owner;
    public BigDecimal sumOfWithdrawals;
    public BigDecimal sumOfDeposits;
    public LocalDate lastTransactionDay;
}

File: CSharp/TestExercises/Bank/AccountReport.cs
using System;

namespace TestExercises.Bank
{
    public class AccountReport
    {
        public AccountReport()
        {
        }

        public Money Balance { get; internal set; }
        public string Owner { get; internal set; }
        public decimal SumOfWithdrawals { get; internal set; }
        public decimal SumOfDeposits { get; internal set; }
        public DateTime LastTransactionDay { get; internal set; }
    }
}